includes/elements/container.php

```diff
@@ -257,7 +263,16 @@ public function before_render() {
 	 * @return void
 	 */
 	public function after_render() {
-		?></<?php $this->print_html_tag(); ?>><?php
+		$settings = $this->get_settings_for_display();
+		?><?php $this->render_inner_wrapper( $settings, 'close' ); ?></<?php $this->print_html_tag(); ?>><?php
+	}
+
+	private function render_inner_wrapper( $settings, $context ) {
+		if ( ! isset( $settings['content_width'] ) || 'boxed' !== $settings['content_width'] ) {
+			return '';
+		}
+
+		echo 'open' === $context ? '<div class="e-container__inner">' : '</div>';
```

Comment:
I'm not sure if I like it

Reply:
@StyleShit 

Any suggestions?

Reply:
I agree with Evyatar.
I don't think there's a need for the `render_inner_wrapper()` method... I would just do the check in each of them. In `after_render()` you already have the `$settings` array available.

Reply:
@StyleShit @udidol Updated.